### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wanda-carlson**
Start

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Can you check it @bert-griffin

**Comment by bert-griffin** [comment]
Hi

**Review by wanda-carlson — COMMENTED** [review]
Can you check this **Bert Griffin**

**Comment by chad-daniels** [comment]
hi

File: README.md [hunk 1]
```diff
@@ -10,3 +10,4 @@ This is a public repository managed by PaidPaper. The code is based in Python. I
 Hi
 Hi, test July 14
 Changes
+Pull request Hurray!
```
Comment (bert-griffin) [inline README.md:13]: Please change

**Review by bert-griffin — CHANGES_REQUESTED** [review]
Changes requested @wanda-carlson

**Pull request closed without merging**